halibot/halibot.py

```diff
@@ -5,7 +5,8 @@
 import json
 import threading
 import os, sys
-from importlib.machinery import SourceFileLoader
+import importlib
+import halibot.packages
```

Comment:
Do we need to modify `sys.path` for this at all, or are we to assume that `halibot` exists already in the python lib path?

Reply:
Just tested - works as expecting. Not sure if our tests use the local directory, but I still do in my own tests. Still works that way, which is nice for lazy me.